src/App.js

```diff
@@ -1,21 +1,34 @@
 import React from 'react';
+import { BrowserRouter as Router, Switch, Route, Link } from 'react-router-dom';
 
 import logo from './logo.svg';
 import './App.css';
+import Login from './screens/Login';
+import Signup from './screens/Signup';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a className="App-link" href="https://reactjs.org" target="_blank" rel="noopener noreferrer">
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Router>
+      <div className="App">
+        <header className="App-header">
+          <img src={logo} className="App-logo" alt="logo" />
+          <nav className="row">
+            <Link to="/">Home</Link>
+            <Link to="/sign_up" className="m-left-5">
+              Signup
+            </Link>
+          </nav>
+        </header>
```

Comment:
Create a component for this

Reply:
In this PR the component for this was created
https://github.com/wolox-training/rs-react-express/pull/2